cypress/integration/alerts_dashboard_flyout_spec.js

```diff
@@ -0,0 +1,287 @@
+      );
+
+      // Confirm 'Monitor' sections renders as expected.
+      cy.get(`[data-test-subj="alertsDashboardFlyout_monitor_${BUCKET_TRIGGER}"]`).contains(
+        'Monitor'
+      );
+      cy.get(`[data-test-subj="alertsDashboardFlyout_monitor_${BUCKET_TRIGGER}"]`).contains(
+        BUCKET_MONITOR
+      );
+
+      // Confirm 'Conditions' sections renders as expected.
+      cy.get(`[data-test-subj="alertsDashboardFlyout_conditions_${BUCKET_TRIGGER}"]`).contains(
+        'Conditions'
+      );
+
+      // Confirm the 'Conditions' sections renders with all of the expected conditions.
+      ['params._count < 10000', 'OR', 'params.avg_products_price == 10'].forEach((entry) =>
+        cy
+          .get(`[data-test-subj="alertsDashboardFlyout_conditions_${BUCKET_TRIGGER}"]`)
+          .contains(entry)
+      );
+
+      // Confirm 'Time range for the last' sections renders as expected.
+      cy.get(`[data-test-subj="alertsDashboardFlyout_timeRange_${BUCKET_TRIGGER}"]`).contains(
+        'Time range for the last'
+      );
+      cy.get(`[data-test-subj="alertsDashboardFlyout_timeRange_${BUCKET_TRIGGER}"]`).contains(
+        '10 day(s)'
+      );
+
+      // Confirm 'Filters' sections renders as expected.
+      cy.get(`[data-test-subj="alertsDashboardFlyout_filters_${BUCKET_TRIGGER}"]`).contains(
+        'Filters'
+      );
+      cy.get(`[data-test-subj="alertsDashboardFlyout_filters_${BUCKET_TRIGGER}"]`).contains(
+        'All fields are included'
+      );
+
+      // Confirm 'Group by' sections renders as expected.
+      cy.get(`[data-test-subj="alertsDashboardFlyout_groupBy_${BUCKET_TRIGGER}"]`).contains(
+        'Group by'
+      );
+      cy.get(`[data-test-subj="alertsDashboardFlyout_groupBy_${BUCKET_TRIGGER}"]`).contains(
+        'customer_gender, user'
+      );
+
+      // Set the 'severity' filter to only display ACTIVE alerts.
+      cy.get('[data-test-subj="dashboardAlertStateFilter"]').select('Active');
+      cy.wait(10000);
+
+      // This monitor configuration consistently returns 46 alerts when testing locally.
+      // Confirm the flyout dashboard contains more than 1 ACTIVE alert.
+      cy.get('tbody > tr').should(($tr) => expect($tr).to.have.length.greaterThan(1));
+
+      // Select the first and last alerts in the table.
+      cy.get('input[data-test-subj^="checkboxSelectRow-"]').first().click();
+      cy.get('input[data-test-subj^="checkboxSelectRow-"]').last().click();
+
+      // Press the flyout 'Acknowledge button, and wait for the AcknowledgeAlerts API call to complete.
+      cy.get('[data-test-subj="flyoutAcknowledgeAlertsButton"]').click();
+    });
+
+    // Confirm acknowledge alerts toast displays expected text.
+    cy.contains('Successfully acknowledged 2 alerts.');
+
+    // Confirm alerts were acknowledged as expected.
+    cy.get(`[data-test-subj="alertsDashboardFlyout_${BUCKET_TRIGGER}"]`).within(() => {
+      // Wait for GetAlerts API call to complete.
+      cy.wait(10000);
```

Comment:
This can probably removed since the next step is only setting filter

Reply:
Removed this `wait` period.